src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -169,12 +173,6 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
             ModifierUtils.CheckAccessibility(this.DeclarationModifiers, this, isExplicitInterfaceImplementation, diagnostics, location);
 
             this.CheckModifiers(location, hasBody: true, isAutoPropertyOrExpressionBodied: true, diagnostics: diagnostics);
-
-            if (syntax != null)
-            {
-                var messageId = property.IsIndexer ? MessageID.IDS_FeatureExpressionBodiedIndexer : MessageID.IDS_FeatureExpressionBodiedProperty;
-                messageId.CheckFeatureAvailability(diagnostics, syntax, syntax.ArrowToken.GetLocation());
-            }
```

Comment:
Should the `IDS_FeaturePropertyAccessorMods` check be moved here (primary constructor for `SourcePropertyAccessorSymbol`) then?

Reply:
moved!